assets/queries/cloudFormation/clientsecret_is_exposed/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "clientsecret_is_exposed",
+  "queryName": "ClientSecret is Exposed",
```

Comment:
Change **directory name** and **id** to: `alexa_skill_plaintext_clientsecret`
Change **query name** to: `Alexa Skill Plaintext Clientsecret`